standup/README.md

```diff
@@ -0,0 +1,44 @@
+# Firestore Standup collection data model
+
+```json
+{
+  "id": "String",          // Unique identifier for the document
+  "type": "String",        // Type of the document, e.g., "task progress" or "user progress"
+  "userId": "String",      // User ID (applicable for both task progress and user progress)
+  "taskId": "String",      // Task ID (applicable for task only)
+  "completed": "String",   // Completed portion of the task (applicable for both similar to yesterday or past progress for user)
+  "planned": "String",     // Planned portion of the task (applicable for both similar to today or upcoming plan for user)
+  "blockers": "String",    // Blockers (applicable for both task and user progress)
+  "date": "Timestamp",     // Date of the standup or user progress (applicable for user progress)
+  "createdAt": "Timestamp" // Creation timestamp (applicable for both)
```

Comment:
if we already have a date field do we need this?
Cant we use date as datetime?

Reply:
`if we already have a date field do we need this?`
since the user can send standup for the day before 6 am of the next day. I think it will be better if we have a day otherwise we need to compute every time based on the created at value whether its for the same day or day before.

Reply:
`Cant we use date as datetime?` 
if we prioritize human-readability and compatibility, ISO 8601 format may be a better choice. But, if we need efficient sorting and comparison operations, Unix timestamp format are more suitable. Let me know your thoughts. Since the super user may be interested in getting updates from a specific time period I decided to go ahead with unix timestamp. let me know your thoughts.